EIPS/payoutable-token-draft.md

```diff
@@ -0,0 +1,302 @@
+    enum PayoutStatusCode {
+        Nonexistent,
+        Ordered,
+        InProcess,
+        FundsInSuspense,
+        Executed,
+        Rejected,
+        Cancelled
+    }
+    function approveToOrderPayout(address orderer) external returns (bool);
+    function revokeApprovalToOrderPayout(address orderer) external returns (bool);
+    function orderPayout(string calldata operationId, uint256 value, string calldata instructions) external returns (bool);
+    function orderPayoutFrom(string calldata operationId, address walletToBePaidOut, uint256 value, string calldata instructions) external returns (bool);
+    function cancelPayout(string calldata operationId) external returns (bool);
+    function processPayout(address orderer, string calldata operationId) external returns (bool);
+    function putFundsInSuspenseInPayout(address orderer, string calldata operationId) external returns (bool);
+    function executePayout(address orderer, string calldata operationId) external returns (bool);
+    function rejectPayout(address orderer, string calldata operationId, string calldata reason) external returns (bool);
+
+    function isApprovedToOrderPayout(address walletToDebit, address orderer) external view returns (bool);
+    function retrievePayoutData(address orderer, string calldata operationId) external view returns (address walletToDebit, uint256 value, string memory instructions, PayoutStatusCode status);
+
+    event PayoutOrdered(address indexed orderer, string indexed operationId, address indexed walletToDebit, uint256 value, string instructions);
+    event PayoutInProcess(address indexed orderer, string indexed operationId);
+    event PayoutFundsInSuspense(address indexed orderer, string indexed operationId);
+    event PayoutExecuted(address indexed orderer, string indexed operationId);
+    event PayoutRejected(address indexed orderer, string indexed operationId, string reason);
+    event PayoutCancelled(address indexed orderer, string indexed operationId);
+    event ApprovalToOrderPayout(address indexed walletToBePaidOut, address indexed orderer);
+    event RevokeApprovalToOrderPayout(address indexed walletToBePaidOut, address indexed orderer);
+}
+
+```
+
+### Functions
+
+#### approveToOrderPayout
+
+Wallet owner, allows a given address to be payout orderer.
+
+| Parameter | Description |
+| ---------|-------------|
+| orderer | The address of the orderer.
+
+#### revokeApprovalToOrderPayout
+
+Wallet owner, Revokes a given address to be payout orderer.
+
+| Parameter | Description |
+| ---------|-------------|
+| orderer | The address of the orderer.
+
+#### orderPayout
+
+Creates a payout request, that will be processed by the token operator.All payout requests,orders, will be linked to the orderer, associating operationId to orderer, to avoid global collision between operationIds. On operationId cannot be repeated for a given orderer, but there can be two equal operation id's for distinct orderers.
```

Comment:
There is a space missing before `All payout requests`.
It has to be `An operationId cannot be repeated...`